consultation_analyser/consultations/jinja2/schema.html

```diff
@@ -0,0 +1,123 @@
+{% extends "base.html" %}
+
+{% set page_title = "Consultation data schema" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">Consultation data schema v0.1</h1>
+
+  <p class="govuk-body">This page lists the data components required for a consultation to be suitable for processing by the analyser. </p>
+
+  <p class="govuk-body">Data should be provided in one JSON file containing both the consultation itself and a list of responses. That overall data file is described here and an example is given. Underneath are details of each section, including individual JSON schemas for both the Consultation and ConsultationResponse parts of the JSON file.</p>
+
+  <h2 class="govuk-heading-l">JSON schemas</h2>
+
+  <h3 class="govuk-heading-m">ConsultationWithResponses JSON</h3>
+  <p class="govuk-body">This is the schema for the file the analyser tool will accept, so if your JSON is valid according to this schema, is is ready to upload. For advice on how to do this in Python, see <a class="govuk-link" href="https://python-jsonschema.readthedocs.io/en/latest/validate/">the jsonschema documentation</a>. Other languages will have equivalent libraries.</p>
+
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        JSON schema
+      </span>
+    </summary>
+    <div class="govuk-details__text" data-qa="consultation-with-responses-schema">
+      <p class="govuk-body"><a class="govuk-link" download href="/schema/consultation_with_responses_schema.json">Download the JSON schema</a></p>
+      <pre><code>{{ json_schemas['consultation_with_responses'] }}</code></pre>
+    </div>
+  </details>
+
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        Example JSON
+      </span>
+    </summary>
+    <div class="govuk-details__text" data-qa="consultation-with-responses-example">
+      <pre><code>{{ json_examples['consultation_with_responses'] }}</code></pre>
+    </div>
+  </details>
+
+  <h3 class="govuk-heading-m">Consultation JSON</h3>
+  <p class="govuk-body">The schema for the consultation part of the file, provided in case it is easier to work on this part individually.</p>
+
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        JSON schema
+      </span>
+    </summary>
+    <div class="govuk-details__text" data-qa="consultation-schema">
+      <p class="govuk-body"><a class="govuk-link" download href="/schema/consultation_schemajson">Download the JSON schema</a></p>
```

Comment:
I get errors trying to download the JSON schema for the consultation JSON. (The download for the other 2 schemas works.)
I get `consultation_schemajson.html` - "File wasn't available on site"

Reply:
Oh, I think it's just missing a dot i.e. `/schema/consultation_schema.json`

Reply:
Now updated.